documentation/staging/content/managing-domains/domain-lifecycle/scaling.md

```diff
@@ -16,14 +16,21 @@ WebLogic Server supports two types of clustering configurations, configured and
 
 For more in-depth information on support for scaling WebLogic clusters in Kubernetes, see [WebLogic Dynamic Clusters on Kubernetes](https://blogs.oracle.com/weblogicserver/weblogic-dynamic-clusters-on-kubernetes).
 
+WebLogic Kubernetes Operator 4.0 introduces a new Kubernetes resource, called a Cluster resource, that is of custom resource type `Cluster`.  Cluster resources are monitored by the WebLogic Kubernetes operator. A cluster resource references an individual WebLogic cluster and its configuration.  It also controls how many member servers are running along with potentially any additional Kubernetes resources that are specific to that WebLogic cluster.  The Cluster resource enables the `/scale` subresource which allows it to be scaled up or down via an HTTP endpoint.
+
+For more in-depth information on the scale subresource, see [Scale subresource](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#scale-subresource).
+
 The operator provides several ways to initiate scaling of WebLogic clusters, including:
 
-* [On-demand, updating the Domain directly (using `kubectl`)](#on-demand-updating-the-domain-directly).
+* [On-demand, updating the Domain or Cluster directly (using `kubectl`)](#on-demand-updating-the-domain-or-cluster-directly).
+* [kubectl CLI commands](#kubectl-cli-commands).
 * [Calling the operator's REST scale API, for example, from `curl`](#calling-the-operators-rest-scale-api).
+* [Kubernetes Horizontal Pod Autoscalar (HPA)](#kubernetes-horizontal-pod-autoscalar-hpa).
 * [Using a WLDF policy rule and script action to call the operator's REST scale API](#using-a-wldf-policy-rule-and-script-action-to-call-the-operators-rest-scale-api).
 * [Using a Prometheus alert action to call the operator's REST scale API](#using-a-prometheus-alert-action-to-call-the-operators-rest-scale-api).
 
-#### On-demand, updating the Domain directly
+#### On-demand, updating the Domain or Cluster directly
+##### Updating the Domain directly
 The easiest way to scale a WebLogic cluster in Kubernetes is to simply edit the `replicas` field of a Domain. This can be done by using `kubectl`. More specifically, you can modify the Domain directly by using the `kubectl edit` command.  For example:
```

Comment:
I actually think that `kubectl scale` or `kubectl patch` are preferred because these don't require jumping in to a text editor and so can be scripted. Would you mind moving those options up and then removing the "The easiest way" comment?